tests/test_run_state_async.py

```
@@ -0,0 +1,45 @@
+        "status": "completed",
+        "created_at": "2024-01-01T00:00:00Z",
+    }
+    run2 = {
+        "run_id": "run-2",
+        "flow_id": "flow-b",
+        "status": "pending",
+        "created_at": "2024-01-02T00:00:00Z",
+    }
+
+    # Helper to write run state
+    def create_run_state(run_data):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `create_run_state`

Add return type annotation: `None`

(ANN202)